mod-di-converter-storage-server/src/main/resources/templates/db_scripts/defaultData/default_qm_authority_create_job_profile.sql

```diff
@@ -246,3 +246,33 @@ INSERT INTO ${myuniversity}_${mymodule}.action_to_mapping_profiles (id, jsonb) v
      "detailProfileType": "MAPPING_PROFILE",
      "masterProfileType": "ACTION_PROFILE"
    }') ON CONFLICT DO NOTHING;
+
```

Comment:
I am curious, why was this change was needed?

Reply:
because, association_view is addressing new association table, without it tests are failing